src/frontend/scss/components/templates/courses/cms/_course_detail.scss

```diff
@@ -55,7 +55,7 @@
 
       @include media-breakpoint-up(lg) {
         @include sv-flex(1, 0, $r-course-aside);
-        padding: 3rem 0.5rem;
```

Comment:
I'll add a `padding-right: 3rem` to `.course-detail__content`. In my opinion, course content and course aside elements need to be spaced on desktop.

I think it is a good to add padding right to `course-detaill__content` and not on `.course-detail__aside` to reduce paragraph width which are a little bit too long.

Reply:
Do you mean in place of this modification, or in addition ?

Reply:
In addition